### PR TITLE
Unify checkout mutations/resolvers to use id field.

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it unifies all checkout's mutations and checkout's resolver to use `id`  field instead of `token` or `checkoutID`

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6634 Unify checkout's ID fields](https://app.clickup.com/t/2549495/SALEOR-6634)

File: saleor/graphql/payment/mutations.py [hunk 1]
```diff
@@ -123,13 +122,20 @@ class CheckoutPaymentCreate(BaseMutation, I18nMixin):
...
+        )
+        token = UUID(
+            description=f"Checkout token.{DEPRECATED_IN_3X_INPUT} Use `id` instead.",
+            required=False,
+        )
         checkout_id = graphene.ID(
+            required=False,
             description=(
-                f"The ID of the checkout. {DEPRECATED_IN_3X_INPUT} Use token instead."
+                f"The ID of the checkout. {DEPRECATED_IN_3X_INPUT} " "Use `id` instead."
```
Comment (maarcingebala) [inline saleor/graphql/payment/mutations.py:136]:
```suggestion
                f"The ID of the checkout. {DEPRECATED_IN_3X_INPUT} Use `id` instead."
```

Comment (maarcingebala) [reply saleor/graphql/payment/mutations.py:136]: (Occurs in a few more places).

Comment (korycins) [reply saleor/graphql/payment/mutations.py:136]: Thanks for catching this <3

**Review by fowczarek — APPROVED** [review]

**Review by jakubkuc — APPROVED** [review]

**Review by SzymJ — APPROVED** [review]

**Pull request merged**